packages/components/src/components/Passphrase/EnterOnTrezorButton.tsx

```diff
@@ -0,0 +1,39 @@
+import { FormattedMessage } from 'react-intl';
+import styled from 'styled-components';
+import { Image } from '../Image/Image';
+import { DeviceModelInternal } from '@trezor/connect';
+import { Card } from '../Card/Card';
+import { Icon } from '../assets/Icon/Icon';
+import { spacingsPx } from '@trezor/theme';
+
+const Row = styled.div`
+    display: flex;
+    gap: ${spacingsPx.xs};
+    align-items: center;
+    justify-content: space-between;
+`;
+
+interface EnterOnTrezorButtonProps {
+    submit: (value: string, passphraseOnDevice?: boolean) => void;
+    value: string;
+    deviceModel?: DeviceModelInternal;
+}
+
+export const EnterOnTrezorButton = ({ submit, value, deviceModel }: EnterOnTrezorButtonProps) => {
```

Comment:
Are you sure this belongs to the components package?

Reply:
Definitely not, but we should extract it in different PR. I added it as a future improvement to the DS.